apidoc/Titanium/UI/Notification.yml

```diff
@@ -52,6 +52,11 @@ properties:
     type: Number
     constants: Titanium.UI.NOTIFICATION_DURATION_*
     default: <Titanium.UI.NOTIFICATION_DURATION_SHORT>
+  - name: gravity
+    summary: Determines the location at which the notification should appear on the screen.
+    type: Number
+    constants: Titanium.UI.Android.GRAVITY_*
+    platforms: [android]
```

Comment:
Hey @manumaticx, forgot to finish the review here. Please add the `since: "5.4.0"` here as well and we can merge today. Thank you! 🚀